x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/document_detail_logic.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kea, MakeLogicType } from 'kea';
+import { i18n } from '@kbn/i18n';
+
+import { HttpLogic } from '../../../shared/http';
+import { EngineLogic } from '../engine';
+import { flashAPIErrors, setQueuedSuccessMessage } from '../../../shared/flash_messages';
+import { FieldDetails } from './types';
+
+export interface DocumentDetailLogicValues {
```

Comment:
[nit] no need to export kea actions/values interfaces any longer, since MakeLogicType handles that for us
```suggestion
interface DocumentDetailLogicValues {
```

Reply:
I started exporting this for tests, so that I can add a type to the values we mock. I find this helpful.

Reply:
I'm not sure I'm following, Kea should already be type-checking & validating any mock values we enter - why do we need to re-define them?

Reply:
Yeah you're probably right

505a9d38db4eddd517d4ef5db8c2524e41878491